src/crate/crash/command.py

```diff
@@ -285,6 +285,19 @@ def _connect(self, server):
             self.logger.critical('CONNECT ERROR')
         else:
             self.logger.info('CONNECT OK')
+            schema_name = \
+                "table_schema" if self.connection.lowest_server_version >= \
+                TABLE_SCHEMA_MIN_VERSION else "schema_name"
+
+            information_schema_query = \
+                """ select count(distinct(table_name))
+                        as number_of_tables
+                    from information_schema.tables
+                    where {schema}
+                    not in ('information_schema', 'sys', 'pg_catalog') """
+
+            SysInfoCommand.CLUSTER_INFO.append(
+                information_schema_query.format(schema=schema_name))            
```

Comment:
please move that check to a method like
```
def information_schema_query(self, version):
....
```

and add a unit test for that method